### PR TITLE
refactor: separate kcp and skr watcher

### PR DESCRIPTION
**Pull request by adityabhatia**

### CODE REVIEW

File: skr/go.mod [hunk 1]
```diff
@@ -1,9 +1,12 @@
-module github.com/kyma-project/kyma-watcher/operator
+module github.com/kyma-project/kyma-watcher/skr
 
 go 1.17
 
+replace github.com/kyma-project/kyma-watcher/kcp => ../kcp
```
Comment (jakobmoellersap) [inline skr/go.mod:5]: Do we want to commit this override? Probably this is just for testing, but I still wanted to ask

File: kcp/go.mod [hunk 2]
```diff
@@ -0,0 +1,3 @@
+module github.com/kyma-project/kyma-watcher/kcp
+
+go 1.17
```
Comment (jakobmoellersap) [inline kcp/go.mod:3]: Are we switching to go 1.18 for the project in a future PR?

**Review by jakobmoellersap — APPROVED** [review]

File: skr/go.mod (hunk 1 above)
Comment (adityabhatia) [reply skr/go.mod:5]: Yes we intend to keep it until kcp part is merged.

File: kcp/go.mod (hunk 2 above)
Comment (adityabhatia) [reply kcp/go.mod:3]: For kubebuilder init support I used this version, but I can use 1.18 since we do not need that anymore :)

Comment (adityabhatia) [reply kcp/go.mod:3]: https://github.com/kyma-project/kyma-watcher/pull/19/commits/5756aa332816d177f6d97c79ad4d4b432c84e3bd

**Pull request merged**